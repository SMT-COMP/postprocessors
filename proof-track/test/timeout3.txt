0.00/0.00	; run solver
576.11/527.72	; checking proof
710.50/600.07	EOF
