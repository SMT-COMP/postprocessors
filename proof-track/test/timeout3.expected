starexec-result=starexec-unknown
reason=timeout
